### PR TITLE
Usage patterns

### PR DESCRIPTION
**Pull request by ned1313**
Adding a Usage Patterns section to the docs and starting off with a Service Identities doc.

### CODE REVIEW

File: docs/content/operations/usage_patterns/service_identity.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+The Service Selector is used by both Access and Traffic Policies for destination selection. Both the source and destination must match for a policy to apply. Let's review how the Service Selector works and then how it could apply to both Access and Traffic Policies.
+
+The `ServiceSelector` spec defines two selection mechanisms, `KubeServiceMatcher` and `KubeServiceRefs`. The `KubeServiceRefs` is a list of direct references to Kubernetes services; using the cluster, namespace, and service name as required fields for identification. A simple example of the `reviews` service is shown below:
+
+```yaml
+kubeServiceRefs:
+  - name: reviews
+    namespace: bookinfo
+    cluster: cluster-one
```
Comment (joekelley) [inline docs/content/operations/usage_patterns/service_identity.md:68]:
```suggestion
    clusterName: cluster-one
```

**Review by joekelley — COMMENTED** [review]
just a couple yaml nits

**Comment by ned1313** [comment]
Committed YAML changes from Joe

**Review by joekelley — APPROVED** [review]

**Pull request merged**